modules/cnv-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -0,0 +1,213 @@
+//
+// * cnv/cnv_node_network/cnv-troubleshooting-node-network.adoc
+
+[id="cnv-troubleshooting-incorrect-policy-config_{context}"]
+= Troubleshooting: Incorrect NodeNetworkConfigurationPolicy configuration
+
+Applying a NodeNetworkConfigurationPolicy to the cluster is an easy way to apply changes to the node network configuration consistently across your entire cluster.
+However, this also means it is easy to apply an incorrect configuration.
+
+This troublshooting example investigates how to identify and correct the configuration of a failed network Policy.
+
+In this example, a Linux bridge Policy is applied to an example cluster that has 3 master nodes and 3 worker nodes. The Policy fails to be applied because it references an incorrect interface. The error can be found by investigating the available nmstate resources and corrected by updating the Policy. 
+
+.Procedure
+
+. Create a Policy and apply it to cluster. The following example creates a simple bridge on the `ens01` interface:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ens01-bridge-testfail
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        description: Linux bridge with the wrong port 
+        type: linux-bridge
+        state: up
+        ipv4:
+          dhcp: true
+          enabled: true
+        bridge:
+          options:
+            stp:
+              enabled: false
+          port:
+            - name: ens01
+----
++
+----
+nodenetworkconfigurationpolicy.nmstate.io/ens01-bridge-testfail created
+----
+
+. It can take up to 20 seconds for the policy to apply, after which we can check the status of the Policy to ensure it has been successfully configured:
++
+----
+$ oc get nncp
+----
++
+The response shows that the Policy failed:
++
+----
+NAME                    STATUS
+ens01-bridge-testfail   FailedToConfigure
+----
+
+. The Policy status alone does not tell us if it failed on all nodes or a subset of nodes. Listing the Enactments we can see if the Policy was successful on any of the nodes. If the Policy failed for only a subset it suggests the problem is with specific node configuration; if the Policy failed on all nodes it suggest the problem is with the Policy.
++
+----
+$ oc get nnce
+----
++
+The response shows that the Policy failed on all nodes:
++
+----
+NAME                                   STATUS
+master-1.ens01-bridge-testfail         FailedToConfigure
+master-2.ens01-bridge-testfail         FailedToConfigure
+master-3.ens01-bridge-testfail         FailedToConfigure
+worker-1.ens01-bridge-testfail         FailedToConfigure
+worker-2.ens01-bridge-testfail         FailedToConfigure
+worker-3.ens01-bridge-testfail         FailedToConfigure
+----
+
+. Viewing one of the Enactments we can look at the traceback. To make the output simpler for this example, we'll pipe the output to grep and search for 'error' without case sensitivity:
```

Comment:
"Viewing one of the Enactments we can look at the traceback" is confusing; I think it should be reworded.

ISG is pretty firm on not using "we." Contractions in general ("we'll" being a double whammy) are also no bueno

Reply:
It was part of my happy fluff!
The command had also been changed so I've updated that line: 
View one of the failed Enactments and look at the traceback. The following command uses the output tool `jsonpath` to filter the output: